views/check-your-answers.html

```diff
@@ -9,606 +9,202 @@
 {% endblock %}
 
 {% block content %}
-<div class="govuk-grid-row">
+  <div class="govuk-grid-row" xmlns="http://www.w3.org/1999/html">
```

Comment:
Why this `xml namespace`?

Reply:
It's just what was on the prototype, not sure exactly why it's there though so I'll ask someone